build.js

```diff
@@ -238,33 +237,37 @@ function githubLinks (options) {
 }
 
 // This function builds the layouts folder for all the Stylus files.
-function buildLayouts () {
-  console.log('[metalsmith] build/layouts started')
-  const labelForBuild = '[metalsmith] build/layouts finished'
+function buildCSS () {
+  console.log('[stylus] static/css started')
+  const labelForBuild = '[stylus] static/css finished'
   console.time(labelForBuild)
 
   fs.mkdir(path.join(__dirname, 'build'), () => {
```

Comment:
Did you consider the `{ recursive: true }` option of `fs.mkdir()` to avoid the nested callbacks below? As described by docs;

```
// Creates /tmp/a/apple, regardless of whether `/tmp` and /tmp/a exist.
fs.mkdir('/tmp/a/apple', { recursive: true }, (err) => {
  if (err) throw err;
});
```

Reply:
Yeah, but I thought this change should be made separately after all the other changes have settled :)

Reply:
Also, this requires Node.js 10, another reason I didn't make this change yet.

Reply:
Is that requirement a problem though?

As far as I see it, we're running Node.js 10 in CI and $latest when building on the hosting server [nodejs/build/setup/www/resources/scripts/build-site.sh](https://github.com/nodejs/build/blob/master/setup/www/resources/scripts/build-site.sh#L45).

Reply:
No problem for me, it's just that such change should be a clear patch IMO and not sneaked in in other patches, that is all.